### DISCUSSION TITLE
Logging to Slack Caused a Website Crash when the Slack Service went Down

### DISCUSSION THREAD

**Discussion by lee-stellateam** [laravel/framework#35776] [Q&A]
We are logging to slack through monolog as set in the configuration.  Slack is currently down right now and brought down our website.  Is there a way to avoid this and ignore a CURL connection failure to slack or do we just have to turn off logging to slack all together?

**Comment by lk77** [comment]
i don't think there is a way, this should be fixed in the slack handler i think. You could try to catch it in the Exceptions\Handler.php file but i'm not so sure about it.
